panther

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env php
+<?php
+
+/*
+ * This file is part of the Panther project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Psy\Shell;
+use Symfony\Component\Panther\Client;
+use Symfony\Component\Panther\PantherTestCaseTrait;
+
+require file_exists($autoload = __DIR__.'/vendor/autoload.php') ? $autoload : __DIR__.'/../../autoload.php';
+
+if ($psyshBin = $_SERVER['PANTHER_PSYSH_BIN'] ?? false) {
+    require $psyshBin;
+} elseif (!class_exists(Shell::class)) {
+    fwrite(STDERR, 'PsySH is not installed. Run "composer req psy/psysh".'.PHP_EOL);
+    exit(1);
+}
+
+/**
+ * Starts Panther in a REPL mode.
+ *
+ * @author Dany Maillard <danymaillard93b@gmail.com>
+ */
+
+$repl = new class() {
+    use PantherTestCaseTrait;
+
+    /**
+     * @var Client
+     */
+    public $p;
+
+    public function __construct()
+    {
+        $_SERVER['PANTHER_NO_HEADLESS'] = 1;
```

Comment:
```suggestion
        $_SERVER['PANTHER_NO_HEADLESS'] = $_SERVER['PANTHER_NO_HEADLESS'] ?? 1;
```

Just in case someone has a valid use case for disabling headless mode (i.e: the box has no X servers).

Reply:
Done.